# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/gtest/moz.build
# Commit: cefcb5f4f4e0
# Full Hash: cefcb5f4f4e037cc90d4d87e474020ddac411a7b
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-13 21:42:25
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Only scan wifi when the network changes or when on mobile r=necko-reviewers,kershaw a=reland CLOSED TREE
#   
#   Simplifies the concurrent operations of wifi scanning and reduces the
#   frequency in common cases.  Wifi scanning when on mobile is reduced from
#   every 5 seconds to every minute.  Wifi scans will also happen whenever
# ==============================================================================

diff -r af2b512cb5ac -r cefcb5f4f4e0 netwerk/wifi/gtest/moz.build
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/netwerk/wifi/gtest/moz.build	Sat May 13 00:36:01 2023 +0000
@@ -0,0 +1,13 @@
+# -*- Mode: python; indent-tabs-mode: nil; tab-width: 40 -*-
+# vim: set filetype=python:
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+LOCAL_INCLUDES += ["/netwerk/wifi"]
+
+UNIFIED_SOURCES += [
+    "TestWifiMonitor.cpp",
+]
+
+FINAL_LIBRARY = "xul-gtest"